frontend/src/features/mapSideBar/SidebarContents/subforms/OccupancyForm.tsx

```diff
@@ -47,27 +40,29 @@ export const OccupancyForm: React.FC<IOccupancyProps> = ({
       </Row>
       <InfoSection>
         <Row>
-          <Label required>Total Square Footage</Label>
+          <Label>Total Area</Label>
```

Comment:
This was changed to square meters, so I thought it seemed add to have the title be "square footage"